Gemfile

```diff
@@ -76,8 +49,9 @@ gem 'noid', '~> 0.8'
 
 group :test do
   gem "rspec-its"
-  gem "ci_reporter_rspec"
+  gem "ci_reporter_rspec", require: false
```

Comment:
cispec is a rake task but brakeman is CLI -- both used http://cardiff.library.ualberta.ca/job/HydraNorth